server/applicationset/broadcaster.go

```diff
@@ -0,0 +1,94 @@
+package applicationset
```

Comment:
This file is a copy of server/application/broadcaster.go changing the signature to use the ApplicationSetWatchEvent. The problem is that in this case we have duplicated code which brings a series of maintainability issues. It is better to extract the duplicated logic from both brocasters and centralize the common code.